src/js/components/stock-transfer/StockTransferCheckPage.jsx

```diff
@@ -119,73 +122,19 @@ class StockTransferSecondPage extends Component {
 
   fetchStockTransfer() {
     this.props.showSpinner();
-
-    const url = `/openboxes/api/stockTransfer/${this.props.match.params.id}`;
+    const url = `/openboxes/api/stockTransfers/${this.props.match.params.stockTransferId}`;
```

Comment:
Do we actually need the /openboxes here?

Reply:
In 1.3.9 yes, in 3.3.10 this will be not required